guides/v2.1/cloud/live/stage-prod-migrate.md

```diff
@@ -132,7 +132,7 @@ To migrate static files from your local machine:
 						
 To migrate static files from remote-to-remote environments directly (fast approach):
 
-**Note** In order to transfer media from remote-to-remote environments directly you need to enable ssh agent forwarding, please refer to this [GitHub guidance](https://developer.github.com/v3/guides/using-ssh-agent-forwarding/)
+**Note** In order to transfer media from remote-to-remote environments directly you need to enable ssh agent forwarding, see [GitHub guidance](https://developer.github.com/v3/guides/using-ssh-agent-forwarding/)
```

Comment:
Looks like we need another scrub for use of word "Note" --   use admonition formatting?

Reply:
Yes, but some other time.